### PR TITLE
Dashboard giveaways

### PR DESCRIPTION
**Pull request by nikolatriki**
NT: https://www.notion.so/Implement-Dashboard-Giveaways-Controller-and-Views-4acf47de439e4fd587df8947fefa8986

### CODE REVIEW

File: app/views/giveaways/index.html.erb [hunk 1]
```diff
@@ -24,20 +24,22 @@
...
-                        <small><%= l(giveaway[:created_at], format: :short) %></small><br>
-                        <code class="html"><small><%= giveaway.location %></small></code><br>
-                        <h2><%= giveaway.title.truncate(20)%></h2>
-                        <% unless giveaway.pictures.empty? %>
-                            <div class="img image is-128x128">
-                                <%= image_tag(giveaway.pictures.first.image) %>
-                            </div>
-                        <%end%>
-                    </p>
+                <% unless !giveaway.approved_to.nil?%>
```
Comment (OrceMaceski) [inline app/views/giveaways/index.html.erb:27]: instead of double negation you could use 
`if giveaway.approved_to.nil?` instead

**Comment by OrceMaceski** [comment]
Looking good so far but maybe you could fix the seeds so that you can have multiple users with multiple giveaways/claims and comments that belong to different users on a giveaway

**Review by OrceMaceski — APPROVED** [review]

File: app/views/giveaways/index.html.erb (hunk 1 above)
Comment (nikolatriki) [reply app/views/giveaways/index.html.erb:27]: Yes it's simpler. I will change it.

**Comment by nikolatriki** [comment]
> Looking good so far but maybe you could fix the seeds so that you can have multiple users with multiple giveaways/claims and comments that belong to different users on a giveaway

@OrceMaceski Definitely, that's next. Thanks

**Pull request merged**